### PR TITLE
update cluster status

### PR DESCRIPTION
**Pull request by SunRunAway**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-tidb-cluster.md [hunk 1]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
-| **Creating** | Creating the cluster |
-| **Importing** | The cluster is importing data |
-| **Scaling** | Scaling the TiDB, TiKV, or TiFlash nodes |
-| **Upgrading** | Upgrading the TiDB version |
-| **Unavailable** | The TiDB Cloud service is not available |
-| **Unhealthy** | Part of nodes are unavailable, not enough replicas, and so on |
-| **Recovering** | Backup recovery |
-| **Paused** | The cluster is paused |
-| **Resuming** | Resuming the cluster |
+| **Available** | The cluster is healthy and available. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:18]:
```suggestion
| **AVAILABLE** | The cluster is healthy and available. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 2]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
+| **Creating** | The cluster is being created. The DB cluster is inaccessible while it is being created. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:19]:
```suggestion
| ****CREATING** ** | The cluster is being created. The cluster is inaccessible while it is being created. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 3]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
+| **Importing** | Importing data into the cluster. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:20]:
```suggestion
| **IMPORTING** | Importing data into the cluster. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 4]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
+| **Modifying** | The cluster is being modified. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:21]:
```suggestion
| **MODIFYING** | The cluster is being modified. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 5]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
+| **Unavailable** | The cluster has failed and TiDB can't recover it. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:22]:
```suggestion
| **UNAVAILABLE** | The cluster has failed and TiDB cannot recover it. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 6]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
+| **Paused** | The cluster is paused. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:23]:
```suggestion
| **PAUSED** | The cluster is paused. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 7]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
+| **Resuming** | The cluster is resuming from a pause. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:24]:
```suggestion
| **RESUMING** | The cluster is resuming from a pause. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 8]
```diff
@@ -15,16 +15,14 @@ You can see the current status of each running cluster on the cluster page.
...
+| **Restoring** | The cluster is currently being restored from a backup. |
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:25]:
```suggestion
| **RESTORING** | The cluster is currently being restored from a backup. |
```

**Comment by easonn7** [comment]
LGTM

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 9]
```diff
@@ -71,6 +71,6 @@ To resume a paused cluster, take the following steps:
 
 3. In the dialog, click **Resume** to confirm your choice.
```
Comment (ran-huang) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:72]:
```suggestion
3. In the dialog, click **Resume** to confirm your choice. The cluster status becomes **RESUMING**.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**